crates/txtx-addon-kit/src/types/commands.rs

```diff
@@ -434,10 +434,12 @@ impl Serialize for CommandSpecification {
         S: Serializer,
     {
         let mut ser = serializer.serialize_struct("CommandSpecification", 4)?;
```

Comment:
```suggestion
        let mut ser = serializer.serialize_struct("CommandSpecification", 6)?;
```
This specifies the number of fields in the struct. Though you may want to test before committing this.